compiler/rustc_feature/src/unstable.rs

```diff
@@ -527,6 +527,8 @@ declare_features! (
     (unstable, import_trait_associated_functions, "CURRENT_RUSTC_VERSION", Some(134691)),
     /// Allows associated types in inherent impls.
     (incomplete, inherent_associated_types, "1.52.0", Some(8995)),
+    /// Adds `from_utf8*` functions as inherent methods on the `str` type.
+    (unstable, inherent_str_constructors, "1.82.0", Some(131114)),
```

Comment:
This shouldn't need a compiler feature, just the `#[unstable(feature = ...)]` gates in the library should be sufficient.

Reply:
removed